addons/print/models/res_partner.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from openerp import api, models, fields
+
+
+class res_partner(models.Model):
+
+    _inherit = "res.partner"
+
+    @api.depends('street', 'zip', 'country_id', 'state_id', 'city')
+    def _compute_is_address_valid(self):
```

Comment:
Actually it is not checking the address is valid, i.e. that it does exists. Maybe we should rename the field to has_address, because we won't google map the address to check it exists :p .